src/components/card/contactInfoCard/ContactInfoCard.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { useTheme } from "../../themeProvider/ThemeProvider";
+
+const ContactInfoCard = () => {
+  const darkMode = useTheme();
+
+  return (
+    <div className={`ui fluid link card  ${darkMode ? "inverted" : ""}`}>
+      <div className="ui basic segment">
```

Comment:
you need dark them for segment too ? i m not sure. Just asking

Reply:
no its inheriting @faran4engg